test/functional/p2p_quorum_connect.py

```diff
@@ -190,6 +208,22 @@ def run_test(self):
             time.sleep(10) # wait a bit until the connection gets established
         self.log.info("Completed MN connection probe!")
 
+        ###############################################################################
+        # 5) Now test regular node disconnecting after receiving an auth DMN connection
+        ###############################################################################
+        self.log.info("5) Testing regular node disconnection after receiving an auth DMN connection..")
+        self.disconnect_peers(self.miner)
+        no_version_node = self.miner.add_p2p_connection(TestP2PConn(), send_version=False, wait_for_verack=False)
+        assert_equal(len(self.miner.getpeerinfo()), 1)
+        # send the version as it would be a MN
+        mn_challenge = getrandbits(256)
+        with mn6_node.assert_debug_log(["but we're not a masternode, disconnecting"]):
+            no_version_node.send_message(msg_version(mn_challenge))
+        time.sleep(2)
```

Comment:
this fixes the failure here https://github.com/PIVX-Project/PIVX/runs/4734568472

```suggestion
        with self.miner.assert_debug_log(["but we're not a masternode, disconnecting"]):
            no_version_node.send_message(msg_version(mn_challenge))
            time.sleep(2)
```

Reply:
that was me squashing last minute changes just before going to sleep hehe :v: